WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Options/JetpackRestoreOptionsViewController.swift

```diff
@@ -46,4 +56,20 @@ class JetpackRestoreOptionsViewController: BaseRestoreOptionsViewController {
         self.navigationController?.pushViewController(warningVC, animated: true)
     }
 
+    override func detailActionButtonTapped() {
+        guard let jetpackSettingsURL = URL(string: "https://wordpress.com/settings/jetpack/\(site.displayURL)") else {
+
+            let title = NSLocalizedString("Unable to visit Jetpack settings for site", comment: "Message displayed when visiting the Jetpack settings page fails.")
+            let notice = Notice(title: title)
+            ActionDispatcher.dispatch(NoticeAction.post(notice))
```

Comment:
I learned recently you can just do `displayNotice` on a VC, https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/UIViewController+Notice.swift

```suggestion
            displayNotice(title: title)
```

Reply:
Fixed! 58824fa